simplifier/operations/interface.py

```diff
@@ -123,4 +125,10 @@ def simplify(self, keep_form: bool = False) -> Optional[WorldObject]:
         """
         self._promote_subexpression()
         self._fold_constants()
-        return super().simplify(keep_form)
+        op = super().simplify(keep_form)
+        if isinstance(op, (AssociativeOperation, CommutativeOperation)):
+            operands = op.operands
+            op._promote_single_operand()
+            if len(operands) == 1:
+                return operands[0]
```

Comment:
I think this could be simplified to the following:
```suggestion
            return op._promote_single_operand()
```

Reply:
I've fixed this, thanks!